templates/Angular2Spa/ClientApp/app/app.module.client.ts

```diff
@@ -0,0 +1,21 @@
+import { NgModule } from '@angular/core';
+import { BrowserModule } from '@angular/platform-browser';
+import { FormsModule } from '@angular/forms';
+import { HttpModule } from '@angular/http';
+import { sharedConfig } from './app.module';
```

Comment:
It might make more sense to have sharedConfig be an `NgModule` itself that they import within `imports[]`, since most Apps will have lots of NgModules lazy-loaded etc ?